integration/spark2/src/test/scala/org/apache/carbondata/spark/testsuite/partition/TestAlterPartitionTable.scala

```diff
@@ -779,6 +779,23 @@ class TestAlterPartitionTable extends QueryTest with BeforeAndAfterAll {
          .contains("Data in range info must be the same type with the partition field's type"))
   }
 
+  test("Alter table in or not in default database") {
+    sql("DROP TABLE IF EXISTS carbonTable_default_db")
+    sql("""
+          | create table carbonTable_default_db(id int, name string) partitioned by (city string)
+          | row format delimited fields terminated by ','
```

Comment:
It's a hive table I think, please create the carbon table use stored by 'carbondata' and tblproperties

Reply:
@lionelcao Thank you for your review, I will fix this.